File: leetcode.com/0215_kth_largest_element_in_an_array/solution.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
 var findKthLargest = function(nums, k) {
    var nums = new Float64Array(nums)
    // console.log(nums)
    nums.sort()
    // nums.sort((a, b) => a - b)
    return nums[nums.length - k]
};

File: leetcode.com/0215_kth_largest_element_in_an_array/solution.c
int cmp(int *a, int *b)
{
    return *a - *b;
}

int findKthLargest(int *nums, int numsSize, int k)
{
    qsort(nums, numsSize, sizeof(int), cmp);
    return nums[numsSize - k];
}
